neuron/src/lib/Neuron/Frontend/Zettel/View.hs

```diff
@@ -151,8 +151,9 @@ renderBottomMenu themeDyn mIndexZettel mEditUrl = do
       ffor x $ \case
         Nothing -> blank
         Just indexZettel -> do
-          neuronDynRouteLink (Some . Route_Zettel . Z.zettelSlug <$> indexZettel) ("class" =: "item" <> "title" =: "Home") $
+          neuronDynRouteLink (Some . Route_Zettel . Z.zettelSlug <$> indexZettel) ("class" =: "item" <> "title" =: "Home") $ do
             semanticIcon "home"
+            text " Home"
```

Comment:
Why do we need a space before "Home"? Can you post a screenshot of the result?

Reply:
Sorry, forgot screenshots. I updated the PR description.

I added the space because it looked unbalanced without it. The reason was that semantic-ui examples have space between the icon and label (otherwise the HTML would be unreadable), but reflex won't add space unless you ask.

But investigating the semantic-ui example further, it looks like there is a css rule `.ui.menu .item>i.icon { margin-right: 0.36em; }`. When I inspect the neuron output, there is no such css rule being applied.

Should I change it so that the icon has a right margin, instead of a space?

Reply:
It'd be best to figure out why semantic UI css is not being applied. Is our HTML incorrect, or do we have some other CSS that is overriding it (and if so can that be fixed/removed)?

Also, the #342 is for putting a label in z-index (now called "Impulse"), rather than the home icon (which I don't think we need?).

Reply:
Oh right! Well I wanted to fix it in the Impulse frontend too, but I assumed that the issue was about the bottom buttons, because they are semantic-ui menu items, but it's just a plain bullet list on the Impulse page.